### DISCUSSION TITLE
order_data set as autoload in wp_options

### DISCUSSION THREAD

**Discussion by kulterryan** [woocommerce/woocommerce#48745] [General]
Hi Guys, 
I would like to understand the significance of order_data being autoloaded in a large WooCommerce website.

<img width="1007" alt="Screenshot 2024-06-24 at 9 54 42 AM" src="https://github.com/woocommerce/woocommerce/assets/36154235/3e003f72-88e5-4f67-8a64-e5d73b3cff0a">

**Comment by barryhughes** [comment]
Hi @kulterryan,

Autoloading too many options can cause performance issues, as [documented here](https://developer.wordpress.org/reference/functions/add_option/). I'm a little unsure where these particular options are coming from, though.

Do you see this when using only WooCommerce, on a clean testing site that hasn't inherited data from earlier tests?

**Reply by kulterryan** [reply]
@barryhughes it's happening since I've enabled HPOS with compatibility mode.

**Reply by barryhughes** [reply]
OK, and do you see the same on a clean testing site, running only WooCommerce (with HPOS enabled, and compatibility mode enabled)?

I'm asking because we cannot replicate this, and can't find the relevant code needed to generate these options within the core plugin. Of course, we could be missing something, but another possibility is that this comes from another plugin, your theme, or some custom code.
